GW2EIEvtcParser/EIData/ProfHelpers/Necromancer/NecromancerHelper.cs

```diff
@@ -28,6 +28,7 @@ internal static class NecromancerHelper
             new BuffGainCastFinder(SpectralArmorSkill, SpectralArmorEffect).WithBuilds(GW2Builds.December2018Balance),
             new BuffGainCastFinder(SpectralWalkSkill, SpectralWalkEffectOld).WithBuilds(GW2Builds.StartOfLife, GW2Builds.December2018Balance),
             new BuffGainCastFinder(SpectralWalkSkill, SpectralWalkEffect).WithBuilds(GW2Builds.December2018Balance),
+            new BuffLossCastFinder(PlagueSignetSkill, PlagueSignetEffect),
```

Comment:
This will trigger on downs/deads. It is better to check if an effect exists that matches the cast of the skill. Checking the timestamps of down and dead events can be a bit shaky, the timing threshold will need to be definitively higher than ServerDelayConstant.

Reply:
This would also apply to the current Signet of Midnight detection for Mesmer then. Will have to check whether the unique effects are POV only.

Reply:
That would require the Mesmer going down/dying at the same time as applying 2 seconds of Hide in Shadows to themself, which sounds unlikely if not impossible.

Reply:
Turns out the Signet effect for Necromancer is **not** POV-only. It should be possible to detect other instant cast Signets using their respective effects. What do you think about adding effect-based detection and keeping the current Signet of Midnight detection for logs without effects?

Reply:
They can be effect based, yes. Except the ones giving a unique buff on signet activation (signet of stone, assassin's signet,...).